aif360/sklearn/datasets/openml_datasets.py

```diff
@@ -220,8 +222,11 @@ def fetch_bank(*, data_home=None, cache=True, binary_age=True, percent10=False,
         (45211, 6)
     """
     # TODO: this seems to be an old version
-    df = fetch_openml(data_id=1558 if percent10 else 1461, data_home=data_home
-                      or DATA_HOME_DEFAULT, cache=cache, as_frame=True).frame
+    # df = fetch_openml(data_id=1558 if percent10 else 1461, data_home=data_home
```

Comment:
Can you add/share a test to verifies `df` is the same? Then can remove the commented out lines.

Reply:
@lindawangg  I am not sure if I can add my test to test_datasets.py or someplace else. Please confirm this for me. Apart from that I wish to add this test case for the verification as you mentioned - 

from aif360.sklearn.datasets import (fetch_openML, OpenMLStore)

def test_fetch_openML():
    """Test to verify if the new class OpenMLStore returns the same df as before"""
    store = OpenMLStore()
    df_new = store.download(None, None)
    df_old = df = fetch_openml(data_id=1558 if percent10 else 1461, data_home=data_home
                      or DATA_HOME_DEFAULT, cache=cache, as_frame=True).frame
    assert df_old == df_new

Reply:
Also, should I remove the line(222 old and 224 new) with the TODO thing as well?

Reply:
I'm not the project owner, but i think test_datasets.py is a good place to add this test. https://github.com/Trusted-AI/AIF360/blob/master/tests/sklearn/test_datasets.py#L190C1-L191C1

Could add a new test ie: `test_ml_store`

Reply:
Cool. That is what I was thinking. Will make the changes and push them!